staging/src/k8s.io/kubectl/pkg/cmd/annotate/annotate.go

```diff
@@ -43,10 +43,16 @@ import (
 	"k8s.io/kubectl/pkg/util/templates"
 )
 
+const (
+	MsgNotAnnotated = "not annotated"
+	MsgAnnotated    = "annotated"
+	MsgUnAnnotated  = "unannotated"
+)
+
 // AnnotateOptions have the data required to perform the annotate operation
 type AnnotateOptions struct {
 	PrintFlags *genericclioptions.PrintFlags
-	PrintObj   printers.ResourcePrinterFunc
+	ToPrinter  func(string) (printers.ResourcePrinter, error)
```

Comment:
Why we need from `PrintObj` to `ToPrinter` change?. I guess, we can still use `PrintObj` for this PR's purpose.

Reply:
Updated, PTAL.